MetadataExtractor.Tests/MetadataExtractor.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.3</TargetFramework>
```

Comment:
`netcoreapp` and `netstandard` are different things. `netcoreapp1.3` doesn't exist yet, afaik.